pkg/virt-launcher/virtwrap/util/cpu_utils.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017, 2018 Red Hat, Inc.
+ *
+ */
+
+package util
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/util/hardware"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+func getPodPinnedCpus() ([]int, error) {
+	var cpuset string
+	file, err := os.Open(hardware.CPUSET_PATH)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	if scanner.Scan() {
+		cpuset = scanner.Text()
+	}
+	if err := scanner.Err(); err != nil {
+		return nil, err
+	}
+	cpusList, err := hardware.ParseCPUSetLine(cpuset)
```

Comment:
In general, to be more secure, I would prefer a smart sparse list here. Since the cpusetline is not something the user can set  this should be fine. Let's make sure that we never expand something like this if it comes from a user.